run_integration.py

```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+
+if __name__ == "__main__":
+    if sys.argv[1] and sys.argv[2]:
+        os.environ["C42_USER"] = sys.argv[1]
+        os.environ["C42_PW"] = sys.argv[2]
+        rc = os.system("pytest ./integration -v -rsxX -l --tb=short --strict")
+        sys.exit(rc)
+    else:
```

Comment:
I would explicitly `exit(1)` so that if this ever runs on a pipeline, it will purposely fail

Reply:
Or just raise an exception

`raise Exception("Error: Missing username and password")